tnt/src/main/java/org/dcsa/conformance/standards/tnt/party/TntHelper.java

```diff
@@ -0,0 +1,48 @@
+package org.dcsa.conformance.standards.tnt.party;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import lombok.experimental.UtilityClass;
+
+import java.util.ArrayList;
+import java.util.Set;
+
+@UtilityClass
```

Comment:
This is an experimental feature of Lombok: https://projectlombok.org/features/experimental/UtilityClass
It is changing the `public` methods to `static` and adds a constructor. I am not totally sure if this is a good solution. For myself, I usually add `@NoArgsConstructor(access = lombok.AccessLevel.PRIVATE)` which adds a private constructor and I make the methods `static` myself. 
What do you think?

Reply:
I saw this being used across Conformance framework, hence I have used it.

Reply:
Yes, you can keep it. I just only letting you know my thoughts and doubts about it.